src/components/calendar/top-part-of-calendar.tsx

```diff
@@ -69,18 +71,32 @@ export const TopPartOfCalendar: React.FC<TopPartOfCalendarProps> = ({
   const ticks: ReactChild[] = [];
   for (let i = 0; i < dates.length; i++) {
     const date = dates[i];
+    if(i > 0 && viewMode === "Month" && date.getMonth() === 0) {
     ticks.push(
```

Comment:
What are ticks?

Reply:
ticks are the vertical line in the Gantt chart.